backend/src/main/kotlin/hu/bme/sch/cmsch/config/WebMvcConfig.kt

```diff
@@ -28,6 +28,7 @@ class WebMvcConfig(
         arrayOf("/api/**", "/manifest/**", "/cdn/**").forEach {
             registry.addMapping(it)
                 .allowedOrigins(productionUrl)
+                .allowCredentials(true)
```

Comment:
These are public endpoints, but it shouldn't hurt

Reply:
Yes, but the request fill fail if there are credentials provided